test/syntax.w

```diff
@@ -106,4 +106,23 @@ Whenever Fred moves left {
 
 Whenever Fred waits {
 	Make John say "hurry up" now. 
-}
\ No newline at end of file
+}
+
+Whenever Mary's dog touches dog {
```

Comment:
"dog" in this context would be an object named "dog"?

since it's not touching "a dog"?